_data/redirects.csv

```diff
@@ -16,6 +16,7 @@
 /bubblegumbuy/,http://linaro.co/bubblegum96-buy
 /product/bubblegum96/,https://www.96boards.org/product/bubblegum-96/
 /product/bubblegum96,https://www.96boards.org/product/bubblegum-96/
+https://www.96boards.org/products/ce/bubblegum96/
```

Comment:
@kylekirkby what is this redirecting to?  I see just https://www.96boards.org/products/ce/bubblegum96/